app/views/home/index.html.slim

```diff
@@ -5,9 +5,7 @@
   .col-xs-12
     .page-header
       .head-text.green-text.text-center
-        | <span style="text-decoration:line-through;font-size:90%;">Applications for the 4th edition  will open on 1st of October 2018!</span>
-      .head-text.green-text.text-center
-        | Sorry - There has been a change of plans. Dates for the 4th edition will be confirmed ASAP. Stay tuned!
+        | Applications for the 4th edition will open on 21st of January 2018!
```

Comment:
@pg20 shouldn't that be 2019 ;) I'll fix it

Reply:
@DMips  ahh good catch.. :D I also missed it.